### PR TITLE
Adding me to early adopters

### PR DESCRIPTION
**Pull request by timurista**
## Request to add self to early adopters list
Adding myself to adopters as we are exploring backstage at Roku as request from @stefanalund 

#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [x] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -2,4 +2,5 @@
 | ------------ | ------- | ------------------ |
 | [Spotify](https://www.spotify.com) |[@alund](https://github.com/alund)| Main interface towards all of Spotify's infrastructure and technical documentation.|
 | [bol.com](https://www.bol.com) |[@RoyJacobs](https://github.com/RoyJacobs)| Initial work being done to unify platform tooling.|
-| [DFDS](https://www.dfds.com) |[@carlsendk](https://github.com/carlsendk)| V2 self-service platform.|
\ No newline at end of file
+| [DFDS](https://www.dfds.com) |[@carlsendk](https://github.com/carlsendk)| V2 self-service platform.|
+| [DFDS](https://www.roku.com) |[@timurista](https://github.com/timurista)| Initial work on Cloud engineering service platform.|
```
Comment (benjdlambert) [inline ADOPTERS.md:6]: I think you want this to be [Roku] rather than [DFDS]

Comment (stefanalund) [inline ADOPTERS.md:6]: Copy-paste error here @timurista -- should be Roku

**Review by stefanalund — CHANGES_REQUESTED** [review]

File: ADOPTERS.md [hunk 2]
```diff
@@ -2,4 +2,5 @@
 | ------------ | ------- | ------------------ |
 | [Spotify](https://www.spotify.com) |[@alund](https://github.com/alund)| Main interface towards all of Spotify's infrastructure and technical documentation.|
 | [bol.com](https://www.bol.com) |[@RoyJacobs](https://github.com/RoyJacobs)| Initial work being done to unify platform tooling.|
-| [DFDS](https://www.dfds.com) |[@carlsendk](https://github.com/carlsendk)| V2 self-service platform.|
\ No newline at end of file
+| [DFDS](https://www.dfds.com) |[@carlsendk](https://github.com/carlsendk)| V2 self-service platform.|
+| [Roku](https://www.roku.com) |[@timurista](https://github.com/timurista)| Initial work on Cloud engineering service platform.|
```
Comment (timurista) [inline ADOPTERS.md:6]: @stefanalund yes now fixed. Also, the formatter was overriding this file for me causing some extra dashes to be added and spacing. Not sure if it's prettier settings for md files.

File: ADOPTERS.md (hunk 1 above)
Comment (timurista) [reply ADOPTERS.md]: lol yeah thanks

**Review by stefanalund — APPROVED** [review]

**Pull request merged**